src/test/scala/TestClient.scala

```diff
@@ -21,13 +22,15 @@ object TestClient extends App {
   val logger = Logging(system, getClass)
 
 
-  val file = new File("/Users/jiahuang/3dr/stream-s3-uploader/src/test/resources/recap_res.obj.zip")
+  val file = new File("/Users/jasonmartens/Downloads/Star wars.mp4")
```

Comment:
ehhhhh can we have the test url be something not local? Maybe have it download a large file from the 3dr.public bucket